brodan.biz/index.html

```diff
@@ -92,7 +95,7 @@
       }
       .vertical-center {
         min-height: 100%;  /* Fallback for browsers do NOT support vh unit */
-        min-height: 100vh; /* These two lines are counted as one :-)       */
```

Comment:
Don't do this because it it's not clear you can scroll down. If you want to do this, add an arrow that encourages you to scroll down.